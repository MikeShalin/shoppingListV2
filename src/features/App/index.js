/** @flow **/
import * as React from 'react'
import { Fragment } from 'react'
import {
  Switch,
  Route,
  Redirect,
} from 'react-router-dom'

import Login from 'Features/Login'
import Home from 'Features/Home'
import Registration from 'Features/Registration'

const App = () => (
  <Fragment>
      <Switch>
        <Route path='/' exact to='/' component={Home}/>
        <Route path='/login/' component={Login}/>
        <Route path='/registration/' component={Registration}/>
        <Redirect from='*' to='/'/>
      </Switch>
  </Fragment>
)

export default App
