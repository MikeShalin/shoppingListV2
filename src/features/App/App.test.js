import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import App from './index'

describe('App component', () => {
  it('matches early snapshots', () => {
    const tree = shallow(<App />)
    expect(shallowToJson(tree)).toMatchSnapshot()
  })
})