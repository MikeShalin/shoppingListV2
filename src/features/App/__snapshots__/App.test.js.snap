// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches early snapshots 1`] = `
<Fragment>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
      to="/"
    />
    <Route
      component={[Function]}
      path="/login/"
    />
    <Route
      component={[Function]}
      path="/registration/"
    />
    <Redirect
      from="*"
      push={false}
      to="/"
    />
  </Switch>
</Fragment>
`;
